frontend/src/metabase-lib/metadata/Table.ts

```diff
@@ -146,22 +145,25 @@ class TableInner extends Base {
 
   connectedTables(): Table[] {
     const fks = this.fks || [];
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
What's happening here is very wrong. We're creating `Table` instance from API data that is not processed by the standard `getMetadata` flow. I'll try to fix it in a separate PR. The correct solution would be normalizing and hydrating `origin` and `origin.table` fields.